### ISSUE TITLE
Consistently getting "No results."

### ISSUE THREAD

**Issue by diversemix** [jarun/ddgr#119]
Happy to provide more information - but hopefully there is enough below:

```
$ python --version
Python 3.8.2
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
$ ddgr -v
1.7
$ ddgr bash
No results.
ddgr (? for help) q
$ ddgr --debug --nop  bash
[DEBUG] ddgr version 1.7 Python version 3.8.2
[DEBUG] q:bash, region:us-en, page:0, curindex:1, safe:1
[DEBUG] nextParams:
[DEBUG] proxy:{'https': None}
[DEBUG] ua:ddgr/1.7 (textmode; Linux x86_64; 1024x768)
[DEBUG] Response body written to '/tmp/ddgr-response-knr127i9'.
[DEBUG] Prev nextParams: 
[DEBUG] Next nextParams: 
No results.
$ curl "https://duckduckgo.com/?t=canonical&q=bash&ia=web" 
.... loads of results!
```

**Comment by GoDzM4TT3O** [comment]
Update your DDGR version. You have 1.7, meanwhile 1.9 is already out.

**Comment by diversemix** [comment]
... thanks I installed using `apt` so I am guessing that the ubuntu repositories need updating :+1:

**Comment by diversemix** [comment]
1.9 works a treat - thanks for your help - Happy for this to be closed @GoDzM4TT3O

**Comment by GoDzM4TT3O** [comment]
No problem - v1.9 fixes #115

By the way, you can close the issue yourself, or wait for jarun or a moderator to close it for you (I do not moderate the ddgr project/source)
